contrib/errorprone/src/python/pants/contrib/errorprone/BUILD

```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+contrib_plugin(
+  name='plugin',
+  dependencies=[
+    'contrib/errorprone/src/python/pants/contrib/errorprone/tasks:tasks',
```

Comment:
nit: you could simplify this to just `'contrib/errorprone/src/python/pants/contrib/errorprone/tasks'`, taking advantage of the default target name.